### DISCUSSION TITLE
How to store images as blobs, instead writing into a directory?

### DISCUSSION THREAD

**Discussion by kevinmt24** [pymupdf/pymupdf4llm#57] [Ideas]
- Same as title.

**Comment by JorjMcKie** [comment]
This is a Discussions item primarily - transferring.

**Comment by JorjMcKie** [comment]
Please be more specific and suggest a highlevel design!

**Comment by jason-technology** [comment]
I think the ask is so that we can process the images in-code without having to write them out, then read them back in.
So maybe they would be an array of dataframes.
